kodilla-testing/src/test/java/com/kodilla/testing/shape/ShapeCollectorTestSuite.java

```diff
@@ -28,101 +28,151 @@ public static void afterClass() {
+        Assert.assertEquals(actual, expected);
     }
 
     @Test
-    public void testShowFigures() {
+    public void testShowFiguresSquare() {
         //Given
-        Shape testShape = new Square(4);
-        Shape testShape2 = new Circle(1.0);
-        Shape testShape3 = new Triangle(13.0, 13.0, 24.0);
+        Shape addedShape = new Square(4);
         ArrayList<Shape> testList = new ArrayList<Shape>();
+        String expected = "Square";
+        testList.add(addedShape);
+        //When
+        String actual = testList.get(0).getShapeName();
+        //Then
+        Assert.assertEquals(expected, actual);
+    }
 
+    @Test
+    public void testShowFiguresTriangle() {
+        //Given
+        Shape addedShape = new Triangle(13.0, 13.0, 24.0);
+        ArrayList<Shape> testList = new ArrayList<Shape>();
+        String expected = "Triangle";
+        testList.add(addedShape);
         //When
-        testList.add(testShape);
-        testList.add(testShape2);
-        testList.add(testShape3);
+        String actual = testList.get(0).getShapeName();
+        //Then
+        Assert.assertEquals(expected, actual);
+    }
 
+    @Test
+    public void testShowFiguresCircle() {
+        //Given
+        Shape addedShape = new Circle(1.0);
+        ArrayList<Shape> testList = new ArrayList<Shape>();
+        String expected = "Circle";
+        testList.add(addedShape);
+        //When
+        String actual = testList.get(0).getShapeName();
         //Then
-        Assert.assertEquals(testList.get(0).getShapeName(), "Square");
-        Assert.assertEquals(testList.get(1).getShapeName(), "Circle");
-        Assert.assertEquals(testList.get(2).getShapeName(), "Triangle");
+        Assert.assertEquals(expected, actual);
     }
 
     @Test
-    public void testGetShapeName() {
+    public void testGetShapeNameForSquare() {
+        //Given
+        Shape addedShape = new Square(4);
+        String expected = "Square";
+        //When
+        String actual = addedShape.getShapeName();
+        //Then
+        Assert.assertEquals(expected, actual);
+    }
 
+    @Test
+    public void testGetShapeNameForTriangle() {
+        //Given
+        Shape addedShape = new Triangle(13.0, 13.0, 24.0);
+        String expected = "Triangle";
+        //When
+        String actual = addedShape.getShapeName();
+        //Then
+        Assert.assertEquals(expected, actual);
+    }
 
+    @Test
+    public void testGetShapeNameForCircle() {
         //Given
-        Shape testShape = new Square(4);
-        Shape testShape2 = new Circle(1.0);
-        Shape testShape3 = new Triangle(13.0,13.0,24.0);
+        Shape addedShape = new Circle(1.0);
+        String expected = "Circle";
+        //When
+        String actual = addedShape.getShapeName();
+        //Then
+        Assert.assertEquals(expected, actual);
+    }
 
+    @Test
+    public void testGetFiledForSquare() {
+        //Given
+        final double DELTA = 1e-2;
```

Comment:
Co to jest ta DELTA?

Reply:
http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals%28java.lang.String,%20double,%20double,%20double%29

Myślałem, że dla porządku, zawsze określa się zaokrąglenie jako Delta.